mammoth/opts.py

```diff
@@ -636,10 +616,8 @@ def _add_train_general_opts(parser):
         '--max_generator_batches',
         '-max_generator_batches',
         type=int,
-        default=32,
-        help="Maximum batches of words in a sequence to run "
-        "the generator on in parallel. Higher is faster, but "
-        "uses more memory. Set to 0 to disable.",
+        default=0,
+        help="Deprecated"
```

Comment:
remove deprecated. we don't have to worry about supporting nonexistent previous models